modules/dev-tools/assets/js/deprecation.js

```diff
@@ -68,19 +74,46 @@ export default class Deprecation {
 	 *
 	 * @return {boolean}
 	 */
-	static isSoftDeprecated( version ) {
-		const total = this.compareVersion( version, elementor.config.dev_tools.deprecation.current_version );
+	isSoftDeprecated( version ) {
+		const total = this.compareVersion( version, elementorDevToolsConfig.deprecation.current_version );
 
-		return total <= elementor.config.dev_tools.deprecation.soft_version_count;
+		return total <= elementorDevToolsConfig.deprecation.soft_version_count;
 	}
 
 	/**
 	 * @param {string} version
 	 * @return {boolean}
 	 */
-	static isHardDeprecated( version ) {
-		const total = this.compareVersion( version, elementor.config.dev_tools.deprecation.current_version );
+	isHardDeprecated( version ) {
+		const total = this.compareVersion( version, elementorDevToolsConfig.deprecation.current_version );
+
+		return total >= elementorDevToolsConfig.deprecation.hard_version_count;
+	}
+
+	softDeprecated( name, version, replacement ) {
+		if ( elementorDevToolsConfig.isDebug ) {
+			this.deprecatedMessage( 'soft', name, version, replacement );
+		}
+	}
+
+	hardDeprecated( name, version, replacement ) {
+		this.deprecatedMessage( 'hard', name, version, replacement );
+	}
+
```

Comment:
Extract from the class, and use them as `private` functions.

Reply:
Handled.